src/acsets/amr_conversion/stockflow.py

```diff
@@ -0,0 +1,172 @@
+        auxiliary_dict['expression'] = param_name
+        auxiliaries_list.append(auxiliary_dict)
+
+    idx = 0
+    for flow_id, stock_param_list in params_stock_flow_map.items():
+        for stock_or_param in stock_param_list:
+            link_dict = {'id': 'link' + str(idx + 1)}
+            link_dict['source'] = stock_or_param
+            link_dict['target'] = flow_id
+
+            links_list.append(link_dict)
+            idx += 1
+
+    return {
+        'header': {
+            'name': '',
+            'schema': '',
+            'description': '',
+            'schema_name': 'stockflow',
+            'model_version': '0.1',
+        },
+        'model': {
+            'flows': flows_list,
+            'stocks': stocks_list,
+            'auxiliaries': auxiliaries_list,
+            'links': links_list
+        },
+        'semantics': {'ode': {
+            'parameters': params_list,
+            'initials': initials_list,
+            'observables': [],
+            'time': None
+        }},
+        'metadata': None
+    }
+
+
+def amr_to_acset(amr):
+    """
+    A method that takes in a stock and flow JSON amr dictionary object and outputs an equivalent stock and flow
+    JSON acset dictinary object
+    """
+    flows = amr['model']['flows']
+    stocks = amr['model']['stocks']
+    links = amr['model']['links']
+
+    flows_list = []
+    stocks_list = []
+    links_list = []
+
+    symbols = {}
+    stocks_mapping = {}
+    for parameter in amr['semantics']['ode']['parameters']:
+        if parameter['id'].startswith('p_'):
+            symbols[parameter['id'][2:]] = sympy.Symbol('p.' + parameter['id'][2:])
+
+    for idx, stock in enumerate(stocks):
+        stock_id = idx + 1
+        stock_name = stock['id']
+        stock_dict = {'_id': stock_id, 'sname': stock_name}
+        stocks_list.append(stock_dict)
+        symbols[stock_name] = sympy.Symbol('u.' + stock_name)
+        stocks_mapping[stock_name] = idx + 1
+
+    for idx, flow in enumerate(flows):
+        flow_id = idx + 1
+        upstream_stock = next(filter(lambda stock: stock['sname'] == flow['upstream_stock'], stocks_list)).get(
+            '_id')
+        downstream_stock = next(filter(lambda stock: stock['sname'] == flow['downstream_stock'], stocks_list)).get(
+            '_id')
+        flow_name = flow['name']
+
+        amr_expression_str = flow['rate_expression']
+        acset_expression_str = str(sympy.sympify(amr_expression_str, symbols))
+
+        flow_dict = {}
+
+        flow_dict['_id'] = flow_id
+        flow_dict['u'] = upstream_stock
+        flow_dict['d'] = downstream_stock
+        flow_dict['fname'] = flow_name
+        flow_dict["ϕf"] = acset_expression_str
+
+        flows_list.append(flow_dict)
+
+    link_id = 1
+    for idx, link in enumerate(links):
+        if link['source'] in stocks_mapping:
+            link_dict = {'_id': link_id}
+            link_dict['s'] = stocks_mapping[link['source']]
+            link_dict['t'] = idx
```

Comment:
I originally set `link_dict['t'] = link['target'][4:]` to set the `'t'` field of the output acset to the flow id defined by the input amr link's `'target'` field.